### PR TITLE
Fix issues with Lerna and ng tests

### PR DESCRIPTION
**Pull request by barbados-clemens**
I think I figured out the node modules issue with lerna. 

so lerna boostrap installs the packages in it's own package.json then links what is asked for in the sub projects so it won't install dev deps that aren't defined in the root package.json

at least from my experience. 
so we have now a working version of the angular app that behaves correctly

I also added a nuke and setup npm script to help automate remove node modules and running the set up commands. 

I haven't touched the Nest api project as it seems to behaving for now. but just add the deps to the api local package.json for it to work. _theoretically_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,15 @@
...
+    "api:test": "npm run test --prefix ./packages/api",
     "web": "npm start --prefix ./packages/web",
+    "web:test": "npm run test:w --prefix ./packages/web",
     "format": "lerna run format --parallel && prettier --write *.{json,md,yml} packages/**/*.{json,md,yml}",
     "format:check": "lerna run format:check --parallel && prettier --check *.{json,md,yml} packages/**/*.{json,md,yml}",
     "lerna": "lerna",
-    "check": "npm run lint && npm run format:check && npm run test"
+    "check": "npm run lint && npm run format:check && npm run test",
+    "nuke": "sh nuke-node-modules.sh",
+    "setup": "sh install.sh"
```
Comment (spiray) [inline package.json]: @barbados-clemens Can you update the Readme with this in place of the 2 commands?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+CYAN='\033[1;36m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
```
Comment (spiray) [inline install.sh]: Red and Yellow aren't used, they can be removed

File: nuke-node-modules.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+set -e
+CYAN='\033[1;36m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
```
Comment (spiray) [inline nuke-node-modules.sh]: Red and Yellow are unused.

File: package.json (hunk 1 above)
Comment (barbados-clemens) [reply package.json]: yeah I'll update them.

We'll also need to update those paths for the new app path you're adding in #28

**Comment by barbados-clemens** [comment]
removed those items. Should be good to merge now 👍

**Comment by spiray** [comment]
@barbados-clemens I am thinking that we should have a scripts folder to contain these scripts. We will likely have a few more and I don't want to clutter the root.

**Pull request merged**